Tests/Session Tests/SessionTests.swift

```diff
@@ -812,8 +812,8 @@ class SessionTests: XCTestCase {
             didCompleteExpectation.fulfill()
         }
         let actionData = ActionComponentData(
-            details: RedirectDetails(
-                returnURL: URL(string: "https://google.com")!
+            details: try RedirectDetails(
+                returnURL: URL(string: "https://google.com?redirectResult=some")!
```

Comment:
We can reuse `URL(string: "https://google.com?redirectResult=some")` as `Dummy.returnURL`